clusterdock/topologies/cdh/actions.py

```diff
@@ -38,6 +39,21 @@
 logger.setLevel(logging.INFO)
 
 DEFAULT_CLOUDERA_NAMESPACE = Constants.DEFAULT.cloudera_namespace # pylint: disable=no-member
+KDC_HOST_NAME = 'kdc'
```

Comment:
Can we alphabetize this block of new constants?

Also, let's make the variable `KDC_HOSTNAME` to point out that it's the actual hostname of the node (and not the name of the host, whatever that is).